config/crd/bases/notification.toolkit.fluxcd.io_providers.yaml

```diff
@@ -50,6 +50,16 @@ spec:
                 description: HTTP/S webhook address of this provider
                 pattern: ^(http|https)://
                 type: string
+              certSecretRef:
```

Comment:
Could it be wise to use caCertSecretRef or something at least containing CA? Easily confused with "client cert".

Reply:
Follows standard set by other controllers.
https://toolkit.fluxcd.io/components/image/imagerepositories/